schemas/context/implementationdetails.schema.json

```diff
@@ -0,0 +1,42 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/implementationdetails",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Implementation Details",
+  "type": "object",
+  "meta:extensible": true,
+  "description":
+    "Details about the sdk, library or service used in an application's or web page's implementation of a service.",
+  "definitions": {
+    "implementationdetails": {
+      "properties": {
+        "xdm:name": {
+          "title": "Name",
+          "type": "string",
+          "description": "SDK or endpoint Name.",
+          "meta:enum": {
```

Comment:
The options in `meta:enum` here and in https://github.com/adobe/xdm/pull/153/files#diff-60dc52c2aa81b94815707651bf685e8aR9 are inconsistent.